config/prod/bwhite-beat-aml-bioc-new.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
```

Comment:
There is a new template: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files